packages/server/src/service/alloy.proto.ts

```diff
@@ -39,7 +36,8 @@ const main = async () => {
   await testStatus(inactiveVoter)
   // Only needed to test if ALLOY_RELAXED
   if (process.env.ALLOY_RELAXED) {
-    activeVoter.name = 'Unregistered Voter'
+    activeVoter.firstName = 'Unregistered'
```

Comment:
Can we test the other possible return values from the alloy API?  So 'Cancelled Voter' is a voter who returns 'Cancelled' etc...).